Cargo.toml

```
@@ -10,6 +10,7 @@ repository = "https://github.com/amrbashir/muda"
 documentation = "https://docs.rs/muda"
 categories = ["gui"]
 rust-version = "1.71"
+include = ["README.md", "src/**/*.rs", "Cargo.toml", "LICENSE-APACHE", "LICENSE-MIT"]
```

```suggestion
include = ["README.md", "src/**/*.rs", "Cargo.toml", "LICENSE-APACHE", "LICENSE-MIT", "LICENSE.spdx"]
```
Not really sure if that's required but better safe than sorry when it comes to license stuff 😅

---

Sure, better error on the safe side there.